File: app/src/main/java/com/yawlle/kittymotivation/infra/Phrase.java
package com.yawlle.kittymotivation.infra;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Phrase {
    @SerializedName("en")
    @Expose
    public String en;

    public String getPhrase() {
        return en;
    }

    public void setPhrase() {
        this.en = en;
    }

    @Override
    public String toString() {
        return "Data{" +
                ", title='" + en + '\'' +
                '}';
    }
}

File: app/src/main/java/com/yawlle/kittymotivation/infra/Phrase.kt
package com.yawlle.kittymotivation.infra

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Phrase(
    @Expose
    @SerializedName("author")
    val author: String,
    @Expose
    @SerializedName("en")
    val en: String
)
{
    override fun toString(): String {
            return en + "  " + author
        }
    }
